onnxruntime/core/providers/js/operators/layer_norm.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/providers/js/js_kernel.h"
+
+namespace onnxruntime {
+namespace js {
+
+template <typename T, typename U>
+class LayerNorm : public JsKernel {
+ public:
+  LayerNorm(const OpKernelInfo& info) : JsKernel(info) {
+    ORT_ENFORCE(info.GetAttr("axis", &axis_).IsOK());
+    ORT_ENFORCE(info.GetAttr<float>("epsilon", &epsilon_).IsOK());
+
+    JSEP_INIT_KERNEL_ATTRIBUTE(LayerNormalization, ({
+                                 "axis" : Number($1),
+                                 "epsilon" : Number($2),
+                               }),
+                               static_cast<uint32_t>(axis_),
+                               static_cast<float>(epsilon_));
```

Comment:
from the spec, there is another attribute `stash_type`. I guess this is only for tranning? if we ignore this attribute intentionally, please put a comment there explaining.

Reply:
I've copied behavior of CPU OP from `onnxruntime/core/providers/cpu/nn/layer_norm_impl.h` and it did not have this attribute 🤔